docs/book/src/clusteropenstack/configuration.md

```diff
@@ -42,15 +42,15 @@ Note: You can use [the template file](../templates/cluster-template.yaml) by man
 # Using 'external-cloud-provider' flavor
 clusterctl config cluster capi-quickstart \
   --flavor external-cloud-provider \
-  --kubernetes-version v1.20.6 \
+  --kubernetes-version v1.21.2 \
```

Comment:
maybe we could consider using `v1.21.3`? (capi is still using v1.21.2 I think?)

Reply:
Correct, CAPI is using v1.21.2, that's why I decided to also use this version.
But okay for me to use the latest stable version

Reply:
I upgraded to 1.21.2 in CAPI because that was the latest release at that point. But in CAPI we also try to align the version we use for testing to our documentation. Not sure which version is used for testing in CAPO :)

Reply:
But the main CAPO quickstart doc is in the book anyway. So I guess it makes sense to align to the same version, but it doesn't really matter.

Reply:
@sbueringer the version for testing is also changed in this PR to v1.21.2 according to CAPI :)
I would suggest we not test a newer version as CAPI to avoid edge cases.
WDYT?

Reply:
Ups, overlooked that file.

It's fine to just use v1.21.2. It's pretty safe that v1.21.3 will work too. Imho it doesn't really matter which one you use

Reply:
Okay, if it really doesn't matter if we use a newer (patch) version than CAPI, then I'll change it to the current stable version

Reply:
:+1: for directly going to v1.21.3 . Hopefully we don't hit https://github.com/kubernetes/kubernetes/issues/102676 :-) but let's see (would also be the case for v1.21.2)